flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/reader/KafkaPartitionSplitReaderTest.java

```diff
@@ -175,30 +184,33 @@ public void testPendingRecordsGauge() throws Exception {
         reader.handleSplitsChanges(
                 new SplitsAddition<>(
                         Collections.singletonList(
-                                new KafkaPartitionSplit(new TopicPartition(TOPIC1, 0), 0L))));
+                                new KafkaPartitionSplit(new TopicPartition(topic1Name, 0), 0L))));
         // pendingRecords should have not been registered because of lazily registration
-        assertFalse(metricListener.getGauge(MetricNames.PENDING_RECORDS).isPresent());
+        Assertions.assertFalse(metricListener.getGauge(MetricNames.PENDING_RECORDS).isPresent());
         // Trigger first fetch
         reader.fetch();
         final Optional<Gauge<Long>> pendingRecords =
                 metricListener.getGauge(MetricNames.PENDING_RECORDS);
-        assertTrue(pendingRecords.isPresent());
+        Assertions.assertTrue(pendingRecords.isPresent());
         // Validate pendingRecords
-        assertNotNull(pendingRecords);
-        assertEquals(NUM_RECORDS_PER_PARTITION - 1, (long) pendingRecords.get().getValue());
+        Assertions.assertNotNull(pendingRecords);
+        Assertions.assertEquals(
+                NUM_RECORDS_PER_PARTITION - 1, (long) pendingRecords.get().getValue());
         for (int i = 1; i < NUM_RECORDS_PER_PARTITION; i++) {
             reader.fetch();
-            assertEquals(NUM_RECORDS_PER_PARTITION - i - 1, (long) pendingRecords.get().getValue());
+            Assertions.assertEquals(
+                    NUM_RECORDS_PER_PARTITION - i - 1, (long) pendingRecords.get().getValue());
         }
         // Add another split
         reader.handleSplitsChanges(
                 new SplitsAddition<>(
                         Collections.singletonList(
-                                new KafkaPartitionSplit(new TopicPartition(TOPIC2, 0), 0L))));
+                                new KafkaPartitionSplit(new TopicPartition(topic2Name, 0), 0L))));
         // Validate pendingRecords
         for (int i = 0; i < NUM_RECORDS_PER_PARTITION; i++) {
             reader.fetch();
-            assertEquals(NUM_RECORDS_PER_PARTITION - i - 1, (long) pendingRecords.get().getValue());
+            Assertions.assertEquals(
+                    NUM_RECORDS_PER_PARTITION - i - 1, (long) pendingRecords.get().getValue());
```

Comment:
Nit: all the assert changes could be avoided by using static import. This will help reducing the review effort and minimise any unexpected human errors, e.g. typo.